CHANGELOG/new.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -e
+
+echo "What is this changelog for (kong, kong-ee, kong-manager)?"
+select component in kong kong-ee kong-manager; do
+    case $component in
+        kong|kong-ee|kong-manager) break;;
+        *) echo "Invalid option. Please select again.";;
+    esac
+done
+
+echo
+
+read -p "What is the title of the change? " title
+
+echo
+
+echo "What is the type of the change (feature, bugfix, dependency, deprecation, breaking_change, performance)?"
+select type in feature bugfix dependency deprecation breaking_change performance; do
```

Comment:
The enum of `type` and  `scope` are defined in the `CHANGELOG/schema.json`, Do you think removing hard code(feature|bugfix|dependency|deprecation|breaking_change|performance) here would make things more consistent?

Reply:
Temporary measure, let's discuss later. We need something more reasonable immediately.